deployment/monocular/templates/chartsvc-deployment.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}-chartsvc
+  labels:
+    app: {{ template "fullname" . }}-chartsvc
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  replicas: 1
```

Comment:
BTW, I believe that monocular might have been using more than 1 replica. We should revisit having more than 1 replica all the time before doing the switch.

Reply:
I have created also a task to tackle this in Kubeapps so we can double check that there is no any architectural issue on doing so https://github.com/kubeapps/kubeapps/issues/643

Reply:
True, do you think we should have 3 chartsvc deployments? MongoDB is single-node and with chartsvc relying heavily on that, that's more of a source of failure. That said, it still might make sense. WDYT?

Reply:
> True, do you think we should have 3 chartsvc deployments? MongoDB is single-node and with chartsvc relying heavily on that, that's more of a source of failure

My goal is to not to have 1. 2 is fine. I understand your point about MongoDB but I think that's a bigger issue and actually Monocular is not solving it either yet. Also MongoDB can always be replaced for a managed database while our services aren't.

In the long term I agree that the MongoDB bottleneck needs to be resolved to avoid cascading failures.

Reply:
I believe it's more common to use 3 replicas (odd numbers), but I am fine with either. As for MongoDB, agreed, and I'm looking into whether the Helm Hub can use CosmosDB.